sylvia-derive/src/message.rs

```diff
@@ -830,6 +830,13 @@ impl<'a> MsgVariant<'a> {
     }
 }
 
+// TODO: This could be made msg_type agnostic and stores map of msg_ty/variants.
```

Comment:
Nope. Remove it from here, and create an issue about that. Noone ever tracked TODOs in code comments.